options.go

```diff
@@ -28,11 +28,11 @@ func Headers(headers http.Header) Option {
 	}
 }
 
-// UsingOrganization allows you to apply account-level changes (Load Balancing,
```

Comment:
I'd really like to deprecate this method entirely due to the amount of edge cases and additional code paths it introduces. Ideally explicit use would trump a helper method like this (including in the terraform provider).